operations/auditing/Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+psycopg2 = "*"
```

Comment:
Can you consider {{psycopg2-binary}} instead, so that macos developers can install this?